Bookdown edits (#190)
courtiol/IsoriX: bookdown/Assignment.Rmd

> ---
> editor_options: 
>   markdown: 
>     wrap: sentence
> ---

Reviewer: ```suggestion
```

```

We now detail the variance associated to the test statistic.

> Under the null hypothesis the isotope value at location of origin equals the isotope value at the candidate location, the variance of the test statistic is the variance of the difference between these two isoscape values.

Reviewer: ```suggestion
Under the null hypothesis, we just saw that the isotope value at location of origin equals the isotope value at the candidate location. The variance of the test statistic is thus the variance of the difference between these two isoscape values.
```

### More information on the assignment plots

The plotting method uses the information from the assignment test to define confidence regions.

> By default, a 5%-confidence region is being displayed in grey, which means that the non-grey areas should contain the true location of origin with a 95% probability.

Reviewer: ```suggestion
By default, a region corresponding to the most unlikely 5% is being displayed in grey, which means that the non-grey areas should contain the true location of origin with a 95% probability.
```
